packages/stripe_js/README.md

```diff
@@ -0,0 +1,3 @@
+# Stripe.js bindings for dart2js
+
+This package is in a very early stage so be cautious to use it in production.
```

Comment:
impressive!
can you add a bit of information on how the bindings were generated, as in with what tool + commands?